lib/msf/core/post/file.rb

```diff
@@ -354,21 +354,13 @@ def rm_f(*remote_files)
   # @param old_file [String] Remote file name to move
   # @param new_file [String] The new name for the remote file
   def rename_file(old_file, new_file)
-    if session.respond_to? :commands && session.commands.include?("stdapi_fs_file_move")
+    if session.type == "meterpreter"
```

Comment:
Why this change?  Probably not a huge deal because all current meterpreters implement the file functions, but they didn't always and there's no guarantee that new ones will.